src/NuGet.Clients/NuGet.CommandLine/Commands/PushCommand.cs

```diff
@@ -17,6 +17,12 @@ public class PushCommand : Command
         [Option(typeof(NuGetCommand), "CommandApiKey")]
         public string ApiKey { get; set; }
 
+        [Option(typeof(NuGetCommand), "PushCommandSymbolsSourceDescription")]
+        public string SymbolsSource { get; set; }
```

Comment:
Looking at this again, I think I chose the wrong one. My reasoning was that the file is called .symbols.nupkg so this is the source for that symbols file. BUT... now I see that Visual Studio uses the term Symbol Server instead of Symbols Server. To be consistent I think this should be singular.

---
In reply to: [68445440](https://github.com/NuGet/NuGet.Client/pull/712#discussion_r68445440) [](ancestors = 68445440)